Updated es and es-419 translations

Gudahtt opened the pull request:
Explanation: Updated es and es-419 translations.

Manual testing steps: Just change to Spanish or Spanish (Latin America) language configuration setting.

(This PR was originally created by @fernandospr in #10073, which was accidentally closed)

fernandospr committed:
Updating Spanish messages.json

fernandospr committed:
Updating Spanish (Latin America) messages.json

fernandospr committed:
More updates to Spanish messages.json

fernandospr committed:
More updates to Spanish (Latin America) messages.json

Gudahtt commented:
If any community contributors would like to help review this PR, it would be greatly appreciated!

I'll tag anyone here that has contributed in the past: @Pedro-vk @mapachurro @jonathansmirnoff @raulk @vimalsagar @hermanjunge @omartinex 

I wanted to give you all the opportunity to provide feedback because we owe our current translations to you. Please don't feel obliged if you don't have the time to review, and please let me know if you'd like me to stop tagging you for reviews in the future.

Thanks!

Gudahtt commented:
This PR has been rebased onto `develop` to ensure that the `yourSigRequested` message was not present (it was removed in #10300).

ricardolyn commented on app/_locales/es/messages.json:
```diff
@@ -26,9 +34,18 @@
     "message": "registro de actividades"
   },
+  "addAccount": {
+    "message": "Agregar una cuenta"
```
Maybe I would use more "Añadir una cuenta". need confirmation by 1 more person to see if is worth changing

Gudahtt replied:
I think "Agregar" makes sense here? At least if this comment is accurate: https://www.reddit.com/r/duolingo/comments/1alunr/what_is_the_difference_between_a%C3%B1adir_and_agregar/c8ylfwv/
The "Add" in this case is more the "add to a group/aggregate" case I think. Each account is independent.

ricardolyn commented on app/_locales/es/messages.json:
```diff
@@ -163,32 +281,94 @@
+    "description": "$1 is the number of accounts to which the web3 site/application is asking to connect; this will substitute $1 in connectToMultiple"
+  },
+  "connectWithMetaMask": {
+    "message": "Conectar Con MetaMask"
```
"Conectar con MetaMask"

fernandospr replied:
Yes, however, the [english version](https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L314) is "Connect **W**ith Metamask".

ricardolyn replied:
seems strange but if that is that is how we are using with the brand, then ok.

kumavis replied:
"Connect With Metamask" means the dapp can read your ethereum address

ricardolyn commented on app/_locales/es/messages.json:
```diff
@@ -238,24 +424,56 @@
+    "message": "Descifrar"
+  },
+  "decryptCopy": {
+    "message": "Copiar mensaje cifrado"
```
Shouldn't be instead "Copiar mensaje decifrado"?

fernandospr replied:
I'm not sure about this, the [english version](https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json#L449) is "Copy encrypted message". So I guess "Copiar mensaje cifrado" or "Copiar mensaje encriptado" should be ok.

ricardolyn replied:
got confused by the id then. in that case, is ok.

Gudahtt committed:
Lint fixes

The message `swapGetQuotes` is no longer used, so it has been removed.
Additionally two descriptions were updated to be English. The
descriptions are not meant to be translated.

Gudahtt commented:
I've rebased this PR and made a couple of minor lint fixes.

@ricardolyn Thanks so much for the review! Just wondering - did you finish reviewing the `es` locale? Did you review `es-419` as well?

ricardolyn commented:
yes, I finished. np 👍
